src/map/utils/battleutils.cpp

```diff
@@ -748,6 +748,16 @@ namespace battleutils
         ELEMENT spikeElement = (ELEMENT)((uint8)GetSpikesDamageType(Action->spikesEffect) - (uint8)DAMAGE_TYPE::ELEMENTAL);
         int32   damage       = Action->spikesParam;
 
+        if (PDefender->getMod(Mod::SPIKES_BONUS) > 0)
+        {
+            if (static_cast<SPIKES>(Action->spikesEffect) == SPIKES::SPIKE_BLAZE ||
+                static_cast<SPIKES>(Action->spikesEffect) == SPIKES::SPIKE_ICE ||
+                static_cast<SPIKES>(Action->spikesEffect) == SPIKES::SPIKE_SHOCK)
```

Comment:
Did we implement Deluge Spikes yet? It's castable from some Porrogo cosmetic +1.

Reply:
should it just check if not dread spikes?

Reply:
RDM can't cast dread spikes and the only way to really know would be to use Contagion Transfer from that one shield... maybe we don't need a check at all?